react/components/InputLabel/index.jsx

```diff
@@ -1,5 +1,12 @@
 import React from "react";
 import { InputLabel as MUIInputLabel } from "@mui/material";
+import PropTypes from "prop-types";
+
+Label.propTypes = {
+    required: PropTypes.bool.isRequired,
+    disabled: PropTypes.bool.isRequired,
+    error: PropTypes.bool.isRequired
+}
```

Comment:
Reference before variable initialization silently breaks app, doesn't throw build errors but causes died in status messages